internal/controllers/ping.go

```diff
@@ -21,7 +19,7 @@ func (c ApiController) PingHandler(w http.ResponseWriter, r *http.Request) {
 		log.Errorf("Unable to calculate node downtime, error: %v", err)
 	}
 
-	if math.Abs(downtimeDuration.Seconds()) > pingIntervalSeconds {
+	if math.Abs(downtimeDuration.Seconds()) > stats.PingIntervalInSeconds {
```

Comment:
When you change PingIntervalInSeconds to 5 this should be buffered by some amount because of request time